### PR TITLE
Adding Search functionality

### PR DESCRIPTION
**Pull request by murradkh**

### CODE REVIEW

File: myapp/views.py [hunk 1]
```diff
@@ -26,11 +27,14 @@ def index(request):
...
-            search_query = {"search_title": kwargs['search_title']}
+        if "search_text" in kwargs:
+
+            stocks = Stock.objects.filter(
+                Q(symbol__contains=kwargs['search_text']) | Q(name__contains=kwargs['search_text'])).order_by(
+                '-top_rank')
+            search_query = {"search_text": kwargs['search_text']}
         else:
-            stocks = Stock.objects.all().order_by('top_rank')
+            stocks = Stock.objects.all().order_by('-top_rank')
```
Comment (kamalq97) [inline myapp/views.py:37]: Not sure why you decided to reverse the order of the query

File: myapp/templates/watchlist.html [hunk 2]
```diff
@@ -4,6 +4,7 @@
 <div class="container">
     <div class="col-md-12 text-left">
 		<h3>Your Watchlist</h3>
+</div>
```
Comment (kamalq97) [inline myapp/templates/watchlist.html:7]: Thanks for the quick fix 😉

**Review by kamalq97 — APPROVED** [review]
I can confirm it working fine (I made a minor change to the CSS). It seems like testing for such feature can only be done on the front-end. Maybe that can be added at a later time.

File: myapp/views.py (hunk 1 above)
Comment (murradkh) [reply myapp/views.py:37]: this would list the rank values in descending order.

**Pull request merged**